src/Arcus.Templates.WebApi/Program.cs

```diff
@@ -131,5 +335,68 @@ private static IHostBuilder CreateHostBuilder(string[] args, IConfiguration conf
                 .WriteTo.AzureApplicationInsights(instrumentationKey);
         }
 #endif
+        
```

Comment:
Just a suggestion; these methods are coming from the Startup class, but we do not have a `Startup` any more in .NET 6.
Is it OK to have these methods directly in this file, or should we consider creating a partial class so that we can put these things in a separate file ?
Just thinking out loud.

Reply:
That would be a good idea for maintaining the project. But maybe people do want to maintain this differently, like adding extensions for related functionality. Hmm 🤔, I like the idea but maybe we shouldn't force this on people. Just want to be as comfortable as possible for everyone 😅.

Maybe this can be checked later, some experimentation is maybe in order. 🤔

Reply:
That's a bit the 'problem' with a template, you're always 'forced' in a certain way of working.